config/main.py

```diff
@@ -533,5 +534,15 @@ def ecn(profile, rmax, rmin, ymax, ymin, gmax, gmin, verbose):
     if verbose: command += " -vv"
     run_command(command, display_cmd=verbose)
 
+
+#
+# 'platform' group
+#
+@cli.group()
+def platform():
+    """Platform-related configuration tasks"""
+platform.add_command(mlnx.mlnx)
```

Comment:
will this indentation work?

Reply:
@stcheng It works well, please refer to line 163 in the same file to see how it load 'aaa' command group.